src/service-providers/active_campaign/index.js

```diff
@@ -46,12 +44,18 @@ const renderPreSendInfo = ( newsletterData = {} ) => {
 				) }
 			</p>
 			<p>{ __( 'Are you sure you want to proceed?', 'newspack-newsletters' ) }</p>
-		</Fragment>
+		</>
 	);
 };
 
+const isCampaignSent= ( newsletterData, postStatus = 'draft' ) => {
```

Comment:
```suggestion
const isCampaignSent = ( newsletterData, postStatus = 'draft' ) => {
```

Is our linter broken?

Reply:
Our linter isn't broken, but we decided to temporarily disable the linter from reporting Prettier issues because of the switch to `@wordpress/scripts` (which has different Prettier rules than what were using before) in #1530.